workspace/utilities/lib/picture.xsl

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
+								xmlns:exsl="http://exslt.org/common" 
+								extension-element-prefixes="exsl">
+
+	<xsl:variable name="default-image-sizes">
+		<sizes>
+			<size height="0" width="430" media="(max-width: 430px)" />
```

Comment:
why height 0 ?

Reply:
Yeah that's the part i'm not sure about. It's for the JIT recipe. Check the template `picture-src`

Reply:
I got confuse with the actual height of the image (the height attribute on the `img` element).

Maybe name them `request-height` and `viewport-width` ?

Reply:
I like `request-*`

Reply:
I like `request-*` too !